lib/minesweeper_2pl/game.rb

```diff
@@ -1,4 +1,167 @@
-module Minesweeper_2pl
+module Minesweeper
   class Game
+    attr_accessor :board, :bcli, :game_over
+
+    def initialize(row_size, bomb_count)
+      bcli = BoardCli.new
+      board = Board.new(row_size, bomb_count)
+
+      self.board = board
+      set_board_positions(row_size)
+      self.bcli = bcli
+    end
+
+    def print_board
+      string = bcli.board_to_string(board)
+      bcli.print_message(string)
+    end
+
+    def row_size
+      board.row_size
+    end
+
+    def bomb_count
```

Comment:
I think of "bomb count" as a concept logically related to a `Game` or its `Rules` and of `Board` as a "data container" for the `Game`. Maybe this is a concept that would be more cohesive somewhere else